app/models/team.rb

```diff
@@ -37,6 +40,9 @@ class Team < ApplicationRecord
           -> { lead },
           class_name: TeamProperty.to_s
 
+  has_one :old_team, class_name: 'Team', foreign_key: :moved_to_unit_id
+  belongs_to :moved_to_unit, class_name: 'Team', optional: true
```

Comment:
I think line 44 is a duplicate of line 31?

Reply:
Yikes, will amend